rasa/core/policies/ensemble.py

```diff
@@ -25,6 +25,7 @@
 from rasa.core.exceptions import UnsupportedDialogueModelError
 from rasa.core.featurizers import MaxHistoryTrackerFeaturizer
 from rasa.core.policies.policy import Policy
+from rasa.core.policies.mapping_policy import MappingPolicy
```

Comment:
if we're globally importing this now, can you get rid of the imports from the individual functions?

Reply:
We could have, but this actually seems to be the reason the type checking fails, because when type checking is on the mapping policy imports ensemble and the thing becomes circular. I moved the import into the function and am testing it locally rn

Reply:
it's in the check method now

Reply:
wait what?

Reply:
https://github.com/RasaHQ/rasa/blob/020a6ff705042a381f57edbe02e5876404a4bbb0/rasa/core/policies/mapping_policy.py#L22

imports the ensemble for

https://github.com/RasaHQ/rasa/blob/020a6ff705042a381f57edbe02e5876404a4bbb0/rasa/core/policies/mapping_policy.py#L46

so during type checking this would be circular imports

Reply:
so in the ensemble I moved the mapping policy import into the `check_for_important_policies` method

Reply:
i'm confused, why would a global import for mapping policy not work? That's only the type_checking import for PolicyEnsemble?